xenial/Dockerfile

```diff
@@ -60,4 +60,4 @@ RUN echo 'deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main' > /e
 RUN locale-gen en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US.UTF-8
\ No newline at end of file
+ENV LANGUAGE en_US.UTF-8
```

Comment:
Great, sane line endings, thanks @jancowol. We should show this massimve improvement to @morneode :)